src/lib/button/button.scss

```diff
@@ -89,9 +94,9 @@
   z-index: 1;
 }
 
-// Applies a clearer border for high-contrast mode (a11y)
-@media screen and (-ms-high-contrast: active) {
-  .md-raised-button, .md-fab, .md-mini-fab {
-    border: 1px solid #fff;
+// Add an outline to make it more visible in high contrast mode.
+@include md-high-contrast {
+  [md-button], [md-raised-button], [md-icon-button], [md-fab], [md-mini-fab] {
+    border: solid 1px;
```

Comment:
I tried it out and it looks like using `outline` works, however we can't apply a radius to it. This means that the round buttons would end up looking like this:

![m5veqaz](https://cloud.githubusercontent.com/assets/4450522/20724410/6098406e-b66e-11e6-958d-9045750c9f60.png)